content/en/05-use-case-tutorials/network-monitoring.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Network Monitoring"
+metaTitle: "Using Pixie | Use Cases | Network Monitoring"
+metaDescription: "Network monitoring using Pixie."
+order: 1
+---
+
+This tutorial will demonstrate how to use Pixie to:
+
+- [Visualize the flow of network traffic within your cluster.](#visualize-network-traffic-flowing-within-the-cluster)
```

Comment:
Linking these seems like overkill, especially with the repeated use of "Visualize". it gives the impression that there is going to be a dense amount of content for a 101 tutorial.

Reply:
especially since it already gets a linked outline on the right

Reply:
Hyperlinks are removed. I replaced visualize with graph for slightly less fancy wording.